compiler/onecc-docker/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM ubuntu:18.04 
+
+RUN apt update -y && \ 
+	 apt install -y \ 
+	 wget && \ 
+	 apt clean 
```

```suggestion
RUN apt-get update && apt-get install -qqy --no-install-recommends \
  wget \
  && rm -rf /var/lib/apt/lists/*
```

from https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run